sale_loyalty_order_line_link/readme/CONTRIBUTORS.md

```diff
@@ -3,3 +3,4 @@
   - Pilar Vargas
 - [Ooops404](https://www.ooops404.com):
   - Ilyas
+- \[Heliconia Solutions Pvt. Ltd.\](<https://www.heliconia.io>)
```

Comment:
@HeliconiaSolutions Same as ion antoher PR. Add individual contributor under this.